src/pages/CreditNoteDetails.tsx

```diff
@@ -1132,3 +1204,8 @@ const InlineLink = styled(Link)`
     text-decoration: none;
   }
 `
+
+const InlineButton = styled.span`
```

Comment:
You can use Tailwind classes now! ✨

Reply:
Side note, when I had to create such component I used the Link one from react router rom, having a `#` in `to` but the onclick defined! Maybe that can spare you some cutomization here